src/http/http-driver.ts

```diff
@@ -7,14 +7,20 @@ export class HttpDriver implements Driver {
     private url: URL;
     private authHeader: string | undefined;
 
-    constructor(url: URL) {
+    constructor(url: URL, authToken?: string) {
```

Comment:
Use a `settings` object with a `token` key.
Since TS/JS does not support named parameters, configuration like this is better passed inside an object.

Reply:
Just saw the config object, why not use it all the way?

Reply:
@athoscouto Because `config` has the URL as `string`, but we need it as an `URL` type. And we already parse that in  `createClient`.